plugins/entity-feedback/src/components/FeedbackResponseDialog/FeedbackResponseDialog.tsx

```diff
@@ -130,19 +131,31 @@ export const FeedbackResponseDialog = (props: FeedbackResponseDialogProps) => {
               />
             ))}
           </FormGroup>
+          {Object.keys(responseSelections).every(
+            key => responseSelections[key] === false,
+          ) ? (
+            <FormHelperText error>
+              *select the reason listed above
+            </FormHelperText>
+          ) : null}
         </FormControl>
-        <FormControl fullWidth>
-          <TextField
-            data-testid="feedback-response-dialog-comments-input"
-            disabled={saving}
-            label="Additional comments"
-            multiline
-            minRows={2}
-            onChange={e => setComments(e.target.value)}
-            variant="outlined"
-            value={comments}
-          />
-        </FormControl>
+        {responseSelections.other === true && (
```

Comment:
The `other` response is part of the `defaultFeedbackResponses` but it's not guaranteed to be defined if custom feedback responses are passed in. I don't think we can dynamically hide this based on the response choice selection. What we could do however is allow this comment box to be show/hidden based on an optional prop to this component

Reply:
@kuangp what i understood from above comment is when user clicks on the Incorrect info or missing info we should hide additional comments .right? or you mean something else?

Reply:
What I meant is that `responseSelections` may not always contain the `.other` property. This is only part of the default responses for this component (defined [here](https://github.com/backstage/backstage/blob/master/plugins/entity-feedback/src/components/FeedbackResponseDialog/FeedbackResponseDialog.tsx#L50-L54)), but the consumer of this component may [override](https://github.com/backstage/backstage/blob/master/plugins/entity-feedback/src/components/FeedbackResponseDialog/FeedbackResponseDialog.tsx#L76) the defaults by passing in the `feedbackDialogResponses` prop to this component. Therefore we can't exactly put in logic to depend on the `.other` property. 

What we can do if you want is to define a new component prop that can control whether to hide/show this textbox